include/z64actor.h

```diff
@@ -532,4 +532,31 @@ typedef enum {
 #define UPDBGCHECKINFO_FLAG_6 (1 << 6) // disable water ripples
 #define UPDBGCHECKINFO_FLAG_7 (1 << 7) // alternate wall check?
 
+typedef enum {
+    /* 0x0 */ NPC_TALK_STATE_IDLE, // NPC not currently talking to player
+    /* 0x1 */ NPC_TALK_STATE_TALKING, // NPC is currently talking to player
+    /* 0x2 */ NPC_TALK_STATE_ACTION, // An NPC-defined action triggered in the conversation
+    /* 0x3 */ NPC_TALK_STATE_ITEM_GIVEN // NPC finished giving an item and text box is done
+} NpcTalkState;
+
+typedef enum {
+    /* 0x0 */ NPC_TRACKING_PLAYER_AUTO_TURN, // Determine tracking mode based on player position, see Npc_UpdateAutoTurn
+    /* 0x1 */ NPC_TRACKING_NONE, // Don't track the player
+    /* 0x2 */ NPC_TRACKING_HEAD_AND_TORSO, // Track player by turning the head and the torso
+    /* 0x3 */ NPC_TRACKING_HEAD, // Track player by turning the head
+    /* 0x4 */ NPC_TRACKING_FULL_BODY // Track player by turning the body, torso and head
+} NpcTrackingMode;
+
+typedef struct {
+    /* 0x00 */ s16 talkState;
+    /* 0x02 */ s16 trackingMode;
+    /* 0x04 */ s16 autoTurnTimer;
+    /* 0x06 */ s16 autoTurnState;
+    /* 0x08 */ Vec3s rotHead;
+    /* 0x0E */ Vec3s rotTorso;
+    /* 0x14 */ f32 yPosOffset; // Y position offset to add to actor position when calculating angle to target
+    /* 0x18 */ Vec3f trackPos;
+    /* 0x24 */ s16 pad;
```

Comment:
```suggestion
    /* 0x24 */ char unk_24[0x4];
```
I'm not sure if anything is definitive about the style of unknown/unused struct members, but this is the most common style used

Reply:
I don't know where the s16 came from, it seems to have been changed in #635 even though it's not used anywhere. I'll change it back to the char array :+1: